event-sauce/src/lib.rs

```diff
@@ -302,3 +307,35 @@ where
     /// Delete immediately
     async fn delete(self, store: &S) -> Result<(), S::Error>;
 }
+
+/// A wrapper around a tuple of event and entity, used to purge an entity in the database
+pub struct PurgeBuilder<Ent: Entity, ED: EventData> {
```

Comment:
This should go in the prelude I think.

Reply:
I don't think so, the PurgeBuilder is used internally in `event-sauce` and in the backend storages. If you do a quick search in fluidic's cloudapi for the equivalent `StorageBuilder` you'll see it's not used by name anywhere.

Reply:
Ok, I thought that might be the case with some other items that are exported in the prelude but don't need to be. We can leave that for another PR.